src/Digdir.Domain.Dialogporten.Infrastructure/InfrastructureExtensions.cs

```diff
@@ -100,7 +100,7 @@ public static class InfrastructureExtensions
             // the distributed cache.
             SkipMemoryCache = true,
             // In normal operations, 15 minutes delay is deemed acceptable for authorization data
-            Duration = TimeSpan.FromMinutes(15),
+            Duration = TimeSpan.FromSeconds(1),
```

Comment:
```suggestion
            Duration = TimeSpan.FromMinutes(15),
```

This one should still be 15 min.?